app/utils/load-relationship.ts

```diff
@@ -1,16 +1,17 @@
 import { A } from '@ember/array';
+import OsfModel from 'ember-osf-web/models/osf-model';
 
 interface QueryOptions {
     'page[size]'?: number;
     page?: number;
 }
 
 export default async function loadAll(
-    model,
-    relationship,
+    model: OsfModel,
+    relationship: keyof OsfModel | 'quickfiles',
```

Comment:
(this doesn't really matter because `loadAll` is only used for quickfiles and also should be deleted, but) `keyof OsfModel` doesn't include any relationships, so this is effectively limiting it to `'quickfiles'` and things which will error.

Reply:
lol